x/blob/client/cli/payforblob.go

```diff
@@ -41,13 +47,19 @@ func CmdPayForBlob() *cobra.Command {
 				return fmt.Errorf("failure to create namespace: %w", err)
 			}
 
+			shareVersion, _ := cmd.Flags().GetString(FlagShareVersion)
+			share, err := getShare(shareVersion)
```

Comment:
[nit] Proposal to rename `share` to `shareVersion` because the share type is a distinct concept in celestia-app. See https://github.com/celestiaorg/celestia-app/blob/ab24a7b4d5190a53cf4bb0a9e989f4c824759f8e/pkg/shares/shares.go#L12-L15

```suggestion
			shareVersion, err := getShareVersion(cmd.Flags().GetString(FlagShareVersion))
```